src/be_txn.erl

```diff
@@ -69,10 +74,29 @@ to_json(blockchain_txn_gen_gateway_v1, T, _Ledger) ->
       <<"location">> => ?MAYBE_H3(blockchain_txn_gen_gateway_v1:location(T)),
       <<"nonce">> => blockchain_txn_gen_gateway_v1:nonce(T) };
 to_json(blockchain_txn_routing_v1, T, _Ledger) ->
+
+    ActionJson = fun(Action) ->
+                         case Action of
+                             {update_routers, RouterAddresses} ->
+                                 #{<<"action">> => <<"update_routers">>,
+                                   <<"router_addresses">> =>[?BIN_TO_B58(A) || A <- RouterAddresses]};
```

Comment:
How does this related to "addresses" in the oui txn above? Consolidate?

Reply:
Sorry but consolidate what exactly? For the `routing_txn` there are multiple actions which can be performed, I couldn't think of a way to represent it better in json form. For more context, https://github.com/helium/blockchain-core/pull/434/files#diff-79d16094f5665a5f02f76609f17b0b9cR37 that's the thing we're trying to expose here

Reply:
Sorry, I meant to use the same key for both `router_addresses` and the `addresses` in the oui txn above. It's minor